apps/projects/app/components/Content/IssueDetail/IssueDetail.js

```diff
@@ -315,13 +314,10 @@ const detailsCard = ({
       </Wrapper>
       {issue.workStatus ? <SummaryTable {...summaryData} /> : <Separator />}
       <FieldTitle>Description</FieldTitle>
-      <Text.Block style={{ marginTop: '20px', marginBottom: '20px' }}>
-        <MarkdownWrapper>
-          {issue.body
-            ? renderHTML(marked(issue.body))
-            : 'No description available'}
-        </MarkdownWrapper>
-      </Text.Block>
+      <Markdown
+        content={issue.body}
+        style={{ marginTop: '20px', marginBottom: '20px' }}
```

Comment:
Should We still keep the branch to show a friendly message, in case there is no description?

Reply:
Just checking/documenting: Is this the problem you're referring to?

Previously, we showed this when the description was empty:

![Description: No description available](https://user-images.githubusercontent.com/221614/60724032-59186500-9f03-11e9-94f1-c049e2441348.png)

Now we show this:

![Description:](https://user-images.githubusercontent.com/221614/60724033-59186500-9f03-11e9-8094-c3a8aa0fc07b.png)

Reply:
Fixed.